modules/server/src/main/scala/io/chrisdavenport/ember/server/EmberServerBuilder.scala

```diff
@@ -0,0 +1,128 @@
+package io.chrisdavenport.ember.server
+
+import cats._
+import cats.implicits._
+import cats.effect._
+import fs2.concurrent._
+import org.http4s._
+import org.http4s.server.Server
+import scala.concurrent.duration._
+import java.net.InetSocketAddress
+import java.util.concurrent.Executors
+import java.nio.channels.AsynchronousChannelGroup
+
+
+final class EmberServerBuilder[F[_]: Concurrent: Timer: ContextShift] private (
+  private val host: String,
```

Comment:
The constructor needs to be private, but there's not much harm in exposing read access to the values that need write access.  I wouldn't typically make these vals private.

Reply:
Exposed now